js/data.json

```diff
@@ -1471,5 +1471,25 @@ osts = [
 		"link": "https://www.dropbox.com/s/adhvu2r1646c2j0/Gohan%20ya-san%20Megumi.mp3?dl=1",
 		"name": "Amaama to Inazuma OST - Gohan ya-san Megumi",
 		"img": "https://www.dropbox.com/s/suvtvzcdm5rptxy/amaama%204.jpg?dl=1"
+	},
+	{
+		"link": "https://www.dropbox.com/s/obccd111g0a4ho3/28%20-%20Samidare.mp3?dl=0",
+		"name": "Naruto OST - Samidare",
+		"img": "https://www.dropbox.com/s/c36rmzzdmkvpw54/jiraya_img.jfif?dl=0"
+	},
+	{
```

Comment:
You should move Opening and Ending objects in appropriate arrays, this array is just for OST so leave Naruto here and move others to their respective location.

Reply:
I understand.